src/components/pages/datasets/pep/PepDetail.jsx

```diff
@@ -0,0 +1,248 @@
+import React, { useEffect, useState } from 'react';
+// import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import { dateFormatISO } from 'utils';
+import { ChevronDown, ChevronUp } from 'react-feather';
+import { useParams } from 'react-router-dom';
+import Api from 'api';
+
+
+const PepDetail = () => {
+  const { id } = useParams();
+  const [data, setData] = useState({});
+  const [error, setError] = useState('');
+  const [isDataReady, setDataReady] = useState(false);
+  const [isLoading, setLoading] = useState(false);
+  const { t, i18n } = useTranslation();
+
+  const fetchData = () => {
+    setLoading(true);
+    Api.get(`pep/${id}/`)
+      .then((resp) => {
+        setData(resp.data);
+        setDataReady(true);
+      })
+      .catch((err) => {
+        if (err.response?.data?.detail) {
+          setError(err.response.data.detail);
+        }
+      })
+      .finally(() => {
+        setLoading(false);
+      });
+  };
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+  const [openRelatedPersons, setOpenRelatedPersons] = useState(false);
+  const toggleOpenRelatedPersons = () => {
+    setOpenRelatedPersons(!openRelatedPersons);
+  };
+  const [openRelatedCompanies, setOpenRelatedCompanies] = useState(false);
```

Comment:
all states must be declared at top of component

Reply:
done